src/App.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
+import DailyIframe, { DailyCall, DailyBrowserInfo } from '@daily-co/daily-js';
+import { DailyProvider } from '@daily-co/daily-react';
+import { Link, Outlet, useLocation } from 'react-router-dom';
+
+import { logger } from './utils/Logger.ts';
+
+import { UnsupportedBrowser } from './components/UnsupportedBrowser';
+import { Navigation } from './components/Navigation';
+import './App.css';
+import { DailyLogo } from './components/shared/DailyLogo';
+
+export const App = () => {
+  const [daily, setDaily] = useState<DailyCall | null>(null);
+  const [browserInfo, setBrowserInfo] = useState<DailyBrowserInfo | null>(null);
+
+  useEffect(() => {
+    if (daily) return;
+    const supportedBrowser = DailyIframe.supportedBrowser();
+    setBrowserInfo(supportedBrowser);
+
+    if (supportedBrowser.supported) {
+      logger.info('Browser is supported, going to create the call object!');
+      const instance = DailyIframe.getCallInstance();
+      if (instance) {
+        logger.info(
+          'Browser is supported, but there is already a call object.',
+        );
+        setDaily(instance);
+      } else {
+        const co = DailyIframe.createCallObject();
+        setDaily(co);
+        logger.info('Created call object');
+      }
+    } else {
+      logger.warn('Browser is not supported. Not creating a call object.');
+    }
+  }, [daily]);
+
+  const location = useLocation();
+
+  useEffect(() => {
+    if (!daily || daily.isDestroyed()) return;
+
+    const startCam = async () => {
+      console.log('going to start cam..');
```

Comment:
Maybe stick with the `logger`?
```suggestion
      logger.info('Starting camera…');
```